src/routing_table/network_tests.rs

```diff
@@ -378,6 +379,7 @@ fn groups_have_identical_routing_tables() {
 }
 
 #[test]
+#[ignore]
```

Comment:
As discussed, this should still work, if the test now keeps track of the `expected_peers` per node.